server/routes/patient.js

```diff
@@ -0,0 +1,79 @@
+// patientRoutes
+const express = require('express')
+const patientRoutes = express()
+
+// db models
+const db = require('../../db/')
+const patientModel  = db.model('patient') 
+const planModel = db.model('plan')
+const therapistModel  = db.model('therapist') 
+const treatmentModel  = db.model('patient') 
+const workoutModel  = db.model('therapist') 
+
+// -=-=-= CREATE =-=-=-
+
+// create patient
+patientRoutes.post('/', (req, res, next) => {
+  patientModel.create(req.body)
```

Comment:
Do not pass in `req.body` directly? This is a security vulnerability, we should just map arguments by by key